src/runtime.rs

```diff
@@ -0,0 +1,33 @@
+#[cfg(any(
+    feature = "runtime-async-std",
+    all(
```

Comment:
This any all combo is a little strange?

Reply:
To support simple `cargo test` without specifying any runtime preference

Reply:
Default use async-std

Reply:
I still don't understand why is it
```
async-std OR (!async-std AND !actix AND !tokio)
```
if any runtime is specified, then we use that runtime.
if no runtime is specified, then we simply do nothing?